applications/GeoMechanicsApplication/custom_elements/transient_thermal_element.h

```diff
@@ -168,6 +168,7 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) TransientThermalElement : public Ele
     {
         CheckProperty(DENSITY_WATER);
         CheckProperty(POROSITY);
+        CheckSaturatedLaw();
```

Comment:
It is not necessary just a matter of taste, I would use the same function name for all checks. Here I would use         CheckProperty(RETENTION_LAW, "SaturatedLaw" );